src/server/config/schema.js

```diff
@@ -27,6 +27,21 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
+  cpu: Joi.object({
+    cgroup: Joi.object({
+      path: Joi.object({
```

Comment:
Ah that's unfortunate, but I guess it's intentional to really show that you're going against defaults.